extensions/positron-duckdb/src/extension.ts

```diff
@@ -248,16 +248,50 @@ function makeWhereExpr(rowFilter: RowFilter): string {
 	}
 }
 
+/**
+ * Properly quotes and escapes an identifier for use in DuckDB SQL.
+ * Handles field names containing quotes by doubling them (DuckDB's escaping convention).
+ * Returns a valid SQL identifier even for empty strings.
+ * @param fieldName The field name to quote
+ * @returns The properly quoted and escaped identifier
+ */
 function quoteIdentifier(fieldName: string) {
-	return `"${fieldName}"`;
+	// Handle empty string case by returning a valid SQL identifier
+	if (fieldName === '') {
+		return '""';
+	}
```

Comment:
nit: is this `if` statement redundant?

Reply:
I'll take a closer look and maybe add some tests for the scenario where a column name is an empty string